mne/viz/utils.py

```diff
@@ -38,7 +38,7 @@ def _setup_vmin_vmax(data, vmin, vmax):
             vmin = np.min(data)
         if callable(vmax):
             vmax = vmax(data)
-        elif vmin is None:
+        elif vmax is None:
             vmax = np.max(data)
```

Comment:
I thought this was a bug (and I still think so), but it is in fact the way it is documented in ``plot_tfr_topomap``. Makes no sense to me. Why was it done this way?